lessons/networking/networking-networking-hardware.md

```diff
@@ -1 +1,97 @@
+
+## Broadband Connection
+Broadband connection refers to the high-speed internet access technology which the ISP provides. The fastest broadband connection is fiber which is almost always preferable, but if that is not available then connections such as cable modem and DSL are preferred, wireless and satellite are less reliable, but in cases of rural networks they are the only ones available.  
+
+* Cable Modem
+  * Cable modem service uses coaxial cables that deliver picture sand sound to TV.
+* Digital Subscriber Line (DSL)
+  * DSL service uses traditional copper telephone lines.
+* Fiber
+  * Fiber technology uses light to carry data through transparent glass fibers
+* Wireless
+  * Wireless service uses radio link between home location and nearest service provider facility
+* Satellite
+  * Satellite service is a type of wireless broadband, but the communication is with a satellite rather than a nearby facility
+
+## Wireless Access Point
+Wireless Access Point component of the router provides Wi-Fi access to all of the devices on your local network without the need for Ethernet cables. 
+
+Wi-Fi standard includes two frequency bands 2.4 GHz and 5 GHz. Wi-Fi frequency bands are frequency ranges within the wireless spectrum that are designated to carry Wi-Fi. 2.4 GHz frequency band includes 14 channels, but since they need to be separated to reduce interference only channels 1, 6, and 11 can be used simultaneously without any interference between each other. 5 GHz band has 42 channels where 24 channels are non-overlapping. 
+
+The transmit power of an access point radio is proportional to its effective range. The higher the transmit power, the farther a signal can travel, and the more obstructions it can effectively penetrate.
+
+The advantage of 2.4 GHz band is that it is available at a greater distance from the access point and it is available for a lot of devices, but it is much more congested.
+The advantage of 5 GHz band is that it allows much more bandwidth with much less interference, but it is available at shorter distances and it is not compatible with many older devices. 
+
+## Firewall 
+Home routers also contain a firewall which appropriately protects devices on your LAN from devices on the internet. If you have ever played games on consoles, some of them require modification of the home router firewall in terms of allowing certain ports through or setting up port forwarding which allows devices on the internet to connect to the console on your local LAN. 
+
+Firewalls create rules that allow or deny devices from talking to one another. 
+
+There are two ways of accomplishing that
+* Whitelisting - Everything is denied by default and only things in the rule list are allowed
+* Blacklisting - Everything is allowed by default and only things in the rule list are blocked
+
+# Check YoSelf
+## Q1 - What can make Wi-Fi signal stronger for a device that is further away from the router?
```

Comment:
Please provide four options for multiple choice questions and highlight the correct answer.